README.md

```diff
@@ -13,8 +13,7 @@ part of the compilation process. This covers all use cases, except:
 
 ## Numba support
 
-Numba 0.54.1 and the current master branch are supported - other versions of
-Numba are unsupported.
+Numba 0.54.1 and above are supported.
```

Comment:
Why did this guidance change? Was it last written in between 0.54 and 0.55, or have we started actually testing the intermediate versions 0.55/0.56?

Reply:
When this was written the latest version of Numba was 0.54.1. Since then we've been using ptxcompiler with 0.55, 0.56, and now 0.57, but the guidance never got updated. The intention of the change here is to make the statement clear and also for it to remain accurate for future versions of Numba (assuming nothing else relevant changes).

Reply:
Got it, thanks.